### PR TITLE
Dev small numbers rule

### PR DESCRIPTION
**Pull request by alex-tinarsky**
Added rule to detect writing integers from one to nine as digits instead of words.

### CODE REVIEW

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt [hunk 1]
```diff
@@ -108,8 +108,8 @@ class Checker {
 			smallNumbersRuleBuilder.symbol("$i".toCharArray().first())
 
 			smallNumbersRules.add(smallNumbersRuleBuilder
-				.fromLeft().shouldNotHaveNeighbor(*" \n\t".toCharArray()).getRule() or smallNumbersRuleBuilder
-				.fromRight().shouldNotHaveNeighbor(*" \n\t".toCharArray()).getRule())
+				.fromLeft().shouldNotHaveNeighbor(*" \n".toCharArray()).getRule() or smallNumbersRuleBuilder
+				.fromRight().shouldNotHaveNeighbor(*" \n,;:?$".toCharArray()).getRule())
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt:112]: "-Vous avez dit que vous n’étiez que trois, mais il me semble, à moi, que nous sommes 4." -- extrait de "Trois Mousquetaires"

Suggestion: "ignoringAdjusting(SYMBOLS)"

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt [hunk 2]
```diff
@@ -124,11 +124,11 @@ class SymbolRuleTests: StringSpec({
...
 
 			numberOfRuleViolations += (smallNumbersRuleBuilder
-				.fromLeft().shouldNotHaveNeighbor(*" \n\t".toCharArray()).getRule() or smallNumbersRuleBuilder
-				.fromRight().shouldNotHaveNeighbor(*" \n\t".toCharArray()).getRule())
+				.fromLeft().shouldNotHaveNeighbor(*" \n".toCharArray()).getRule() or smallNumbersRuleBuilder
+				.fromRight().shouldNotHaveNeighbor(*" \n,;:?$".toCharArray()).getRule())
 				.process(PDFBox().getPDF(filePathSmallNumbers)).count()
 		}
-		numberOfRuleViolations shouldBeExactly 1
+		numberOfRuleViolations shouldBeExactly 2
```
Comment (Darderion) [inline src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt:131]: Quotes are not tested
Brackets are not tested
Footnotes are not tested

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt [hunk 3]
```diff
@@ -97,6 +99,19 @@ class Checker {
 			.called("Неправильное использование открывающей кавычки")
 			.getRule()
 
+		val smallNumbersRules = mutableListOf<SymbolRule>()
+		val smallNumbersRuleBuilder = SymbolRuleBuilder()
+			.called("Неправильное написание целых чисел от 1 до 9")
+			.inArea(EVERYWHERE.except(FOOTNOTE, PAGE_INDEX, TABLE_OF_CONTENT))
+
+		for(i in 1..9){
+			smallNumbersRuleBuilder.symbol("$i".toCharArray().first())
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt:108]: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/digit-to-char.html

**Review by Darderion — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt (hunk 3 above)
Comment (alex-tinarsky) [reply src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt]: Applied

File: src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt (hunk 2 above)
Comment (alex-tinarsky) [reply src/test/kotlin/com/github/darderion/mundaneassignmentpolice/checker/SymbolRuleTests.kt]: All of this is being tested now.

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt (hunk 1 above)
Comment (alex-tinarsky) [reply src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt]: Fixed

**Pull request merged**